### PR TITLE
Add: file_make_mustache bundle to render mustache templates

### PR DESCRIPTION
**Pull request by nickanderson**

### CODE REVIEW

File: lib/files.cf [hunk 1]
```diff
@@ -2165,6 +2165,33 @@ bundle agent file_make_mog(file, str, mode, owner, group)
...
+# @param data data container to use
+#
+# **Example:**
+#
+# ```cf3
+# vars:
+#   "state" data => datastate();
+#
+# methods:
+#      "" usebundle => file_make_mustache( "/tmp/z.txt", "/tmp/z.mustache" @(state) );
```
Comment (tzz) [inline lib/files.cf]: typo: missing comma

**Comment by tzz** [comment]
I would also add a bundle to do this with permissions and ownership (since bundles can't take optional parameters).

**Comment by nickanderson** [comment]
Thanks for the review @tzz

**Comment by tzz** [comment]
nice :)

**Pull request merged**